database/migrations/2021_08_20_155305_create_pivot_country_sport_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreatePivotCountrySportTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('country_sport', function (Blueprint $table) {
+           $table->foreignId('country_id')->references('id')->on('countries');
+           $table->foreignId('sport_id')->references('id')->on('sports');
+           $table->foreignId('first')->references('id')->on('countries')->nullable();
```

Comment:
@kjarya I don't think it's a good structure, so you're saving the country in `country_id`, and then ALSO three countries for first/second/third? It's better to save `position` instead, see my #16